kibana-reports/server/utils/validationHelper.ts

```diff
@@ -109,16 +109,30 @@ const validateSavedObject = async (
         return 'search';
       case REPORT_TYPE.visualization:
         return 'visualization';
+      case REPORT_TYPE.notebook:
+        return 'notebook';
     }
   };
 
-  const savedObjectId = `${getType(source)}:${getId(url)}`;
-  const params: RequestParams.Exists = {
-    index: '.kibana',
-    id: savedObjectId,
-  };
+  let exist = false;
+  let savedObjectId = '';
+  if (getType(source) === 'notebook') {
+    const notebookId = getId(url);
+    const params: RequestParams.Exists = {
+      index: '.opendistro-notebooks',
+      id: notebookId
+    }
+    exist = await client.callAsCurrentUser('exists', params);
```

Comment:
does this work with security plugin? I thought `.opendistro-notebooks` is only accessible through `es-notebook`

Reply:
Not sure if this will work with security plugin- but cluster I am running against has es-notebooks installed

Reply:
probably should call Kibana APIs from notebooks or es-notebooks plugin endpoint, calling elasticsearch might not work if you have security enabled. might need to test this with security plugin

Reply:
+1 this needs to be tested